plugins/flp_import/CMakeLists.txt

```diff
@@ -3,4 +3,8 @@ INCLUDE(BuildPlugin)
 INCLUDE_DIRECTORIES(unrtf)
 
 ADD_DEFINITIONS(--std=c++0x)
+IF(HOMEBREW)
+	ADD_DEFINITIONS(-stdlib=libc++)
```

Comment:
Well... no luck with `libstdc++` either and this `ADD_DEFINITIONS` command isn't working, so back to the `${CMAKE_CXX_FLAGS}` logic.

Green lights at least... 

![screen shot 2015-08-16 at 11 09 19 pm](https://cloud.githubusercontent.com/assets/6345473/9297343/d9da992a-446b-11e5-9e85-02299a51599a.png)